data/talks/2018/1/command-and-emission-control.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Command and (e)mission Control",
+  "abstract": "Using event dispatchers and command buses in applications is becoming common place, but are we using them in a way that aids understanding? With the rise in popularity of Domain-Driven Design to drive meaning in applications, and our applications becoming increasingly small and \"hexagonal\", the concept of defining the difference between an Event and a Command is becoming more important. \n\nThis talk explores using command buses and event dispatchers in combination to clearly segregate the structure, and uses an example of how to use these to create clean CQRS-based applications.",
+  "event": "new-year-new-topics",
+  "speakers": [
+    "barney-hanlon"
```

Comment:
Do we have stuff generating speakers yet as I would guess Barney should have a new entry

Reply:
We do have a thing to generate "people", who are either "organisers" or "speakers" (but not "attendees") - see the readme for how to. 

I generated Barney's file yesterday - why do we need a new entry?

Reply:
sorry, me being lazy and I didn't check - was just expecting it to be added with the rest of the January stuff as it would be his first talk with us.